app/models/form.rb

```diff
@@ -1,4 +1,5 @@
 class Form < ActiveRecord::Base
+  has_paper_trail on: [:update, :destroy]
```

Comment:
It makes it a lot easier to revert them sometime down the line if we change them then weeks or months later say "We want to switch them back to what they were" was my only real justification.